### ISSUE TITLE
Connection frontend-backend: Store personal data in database

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#29]
## User story
1. As a {HR employee}
2. I want / need {the information which I enter into the DIdentity frontend to be stored in the database}
3. So that {no information gets lost and the digital identity can be successfully created}

## Acceptance criteria
* The personal data provided via the frontend is stored in the database.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
duplicate with #8 ? I think it was implemented by @Jean28518 and @jackDS008 in 78496d81f2c59621ce2aaf6befe298084bd40feb or a7e53c3fe055f60cb39c6329cc01125f498cd163.
